hathor/transaction/storage/transaction_storage.py

```diff
@@ -541,12 +541,14 @@ def get_transaction(self, hash_bytes: bytes) -> BaseTransaction:
         self.post_get_validation(tx)
         return tx
 
-    def get_block_by_height(self, height: int) -> Optional[Block]:
-        """Return a block in the best blockchain from the height index. This is fast."""
+    def get_block_id_by_height(self, height: int) -> VertexId | None:
```

Comment:
Missing docstring.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575

Reply:
Thanks!